package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubs-client-bot",
-  "version": "0.5.0",
+  "version": "0.8.0",
```

Comment:
Can you revert this whole file? I think some of the additional packages from previous commits got added here, too

Reply:
> Can you revert this whole file? I think some of the additional packages from previous commits got added here, too

Hi! I couldn't find this one. According to me, it is already using version 0.8.0